lighthouse-core/report/templates/report-template.html

```diff
@@ -189,6 +189,8 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   <div class="subitem-result">
                     {{#if subItem.comingSoon}}
                           <span class="subitem-result__unknown score-unknown-bg">N/A</span>
+                    {{else if subItem.error}}
+                      <span class="subitem-result__unknown score-error-bg">N/A</span>
```

Comment:
Red is good b/c it signifies an error, but the icon is different (LH error vs. audit failure) so hopefully people will see the difference. "Audit error" in the debug string should help a lot.